web/src/auth/ForgetPage.js

```diff
@@ -0,0 +1,316 @@
+// Copyright 2021 The casbin Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from "react";
+import { Link } from "react-router-dom";
+import { Button, Col, Form, Input, Row } from "antd";
+import * as AuthBackend from "./AuthBackend";
+import * as ApplicationBackend from "../backend/ApplicationBackend";
+import * as Util from "./Util";
+import * as Setting from "../Setting";
+import i18next from "i18next";
+import { CountDownInput } from "../component/CountDownInput";
+import * as UserBackend from "../backend/UserBackend";
+import {
+  CheckCircleOutlined,
+  LockOutlined,
+  MailOutlined,
+  UserOutlined,
+} from "@ant-design/icons";
+
+class ForgetPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      classes: props,
+      type: props.type,
+      account: props.account,
+      applicationName:
+          props.applicationName !== undefined
+              ? props.applicationName
+              : props.match === undefined
+              ? null
+              : props.match.params.applicationName,
+      application: null,
+      msg: null,
+      email: "",
+      phone: "",
+      emailCode: "",
+      phoneCode: "",
+      verifyType: "phone", // "email" or "phone"
+    };
+  }
+
+  UNSAFE_componentWillMount() {
+    if (this.state.type === "forgotPassword") {
+      this.getApplication();
+    } else {
+      Util.showMessage("error", i18next.t(`forget:Unknown forgot type: `) + this.state.type);
```

Comment:
Then here should be: 
```javascripts
Util.showMessage("error", i18next.t("forgetPassword:Unknown forgot type:" + this.state.type);
```

I haven't noticed that there is a key `forgetPassword` in i18n, sorry...